packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsModal.tsx

```diff
@@ -281,35 +292,62 @@ export const ItemDetailsModal: React.FC<ItemDetailsModalProps> = ({
       const batchRowIdx = newBatchRows.findIndex(({ id }) => batch.id === id);
       const batchRow = newBatchRows[batchRowIdx];
       if (!batchRow) return null;
-      const currentAllocatedUnits = batchRow.numberOfPacks * batchRow.packSize;
-      const totalAvailableUnits =
+      if (toAllocate < 0) return null;
```

Comment:
this whole bit should really be in a method with a few tests around it. will make an issue for that

Reply:
Yeah you're right had thought about it when I touched it also sorry 😁